assignments/closure.js

```diff
@@ -1,14 +1,39 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function lakeHouse(){
+  let mother = 'mama bear';
+  console.log(mother + ' said stop punching your sister!')
+
+  function lake(){
+    let father = "papa bear";
+    console.log(father + ' reminded me that ' + mother + ' does not know what she is talking about')
+  }
+  lake()
+}
+lakeHouse()
+
 
 /* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
 
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
+  let count = 0;
+  return function(){
+    count = count + 1;
+    return count;
+  }
```

Comment:
Perfect use of closures. Notice how the value of `count` is safe in that only the function we made can modify it. No other process can access that variable and mutate it out from under us.